sem_covid/entrypoints/notebooks/pycaret_pwdb_all_classes.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        train_data.reset_index(inplace=True, drop=True)
</pre><p>I wanted to ask longer before, why do you reset indexes? I believe it is unnecessary to do so, and in case you plan to send the data back to ES, the index is in fact needed.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/sem-covid/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>